app/src/main/res/values/strings.xml

```diff
@@ -877,4 +877,7 @@
     <string name="trending_movies">Trending movies and shows</string>
     <string name="trending_music">Trending music</string>
     <string name="entry_deleted">Entry deleted</string>
+    <string name="player_error_403">HTTP error 403 occurred while playing, likely caused by an IP ban or streaming URL deobfuscation issues</string>
```

Comment:
Please make this non YouTube specific. HTTP 403 issues also happen frequently on Soundcloud due to short expiration. I would use something like this instead:

`Unable to play this stream: HTTP error 403 received from server, likely caused by streaming URL(s) expiration or IP ban`.

For other HTTP errors (4XX and 5XX), we should also provide a better error message `Unable to play this stream: got HTTP error [CODE]`